src/Docker.DotNet/Endpoints/ImageOperations.cs

```diff
@@ -88,6 +93,40 @@ public Task CreateImageAsync(ImagesCreateParameters parameters, Stream imageStre
                 cancellationToken,
                 progress);
         }
+        
+        public Task CreateImageAsync(ImagesCreateParameters parameters, Stream imageStream, AuthConfig authConfig, IDictionary<string, string> headers, IProgress<JSONMessage> progress, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I'm totally fine with this. Can we update the original `CreateImageAsync` to call this with `null` for `headers` so we dont maintain two identical function bodies?

Reply:
Thanks jterry75, I have updated the pull request